frontend/src/components/common/Footer.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  Container,
+  Flex,
+  Link,
+  Text,
+  useMediaQuery,
+  VStack,
+} from "@chakra-ui/react";
+import React from "react";
+
+import { Facebook, Instagram } from "./icons";
+
+const Footer = (): JSX.Element => {
```

Comment:
```suggestion
const Footer = (): JSX.Element => (
```
once we remove that media query hook we can make this an entirely UI element